app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
```

Comment:
- [ ] Optional suggestion

Below implementation is enough for the current_user method

```ruby
def current_user
  User.first
end
```